content/docs/intro/console/accounts-and-organizations/accounts.md

```diff
@@ -21,7 +21,7 @@ following:
 * Manage your profile settings, including your account password, access tokens, and subscriptions
 * [Add an organization]({{< relref "organizations" >}}) backed by Atlassian, GitHub, GitLab, or a SAML
   2.0-compatible identity provider, such as Active Directory, Okta, or G Suite
-* [Manage your projects and stacks]({{< relref "../collaboration/project-and-stack-management" >}})
+* [Manage your projects and stacks]({{< relref "docs/intro/console/collaboration/project-and-stack-management" >}})
```

Comment:
Ah, these need to be `/docs` (root-relative).

Reply:
Thank you!